### PR TITLE
fix(LargeCard): document Gutter

### PR DESCRIPTION
**Pull request by cdaringe**
# problem statement

- gutter isn't documented

# solution

- document it, test it

### CODE REVIEW

File: src/components/native/LargeCard/LargeCardGutter.jsx [hunk 1]
```diff
@@ -25,7 +32,7 @@ const LargeCardGutter = (props) => {
   return (
     <Flexbox
       {...externalAttributes}
-      className={`fullview__left_border ${props.className}`}
+      className={`fullview__left_border ${props.className || ''}`}
```
Comment (cdaringe) [inline src/components/native/LargeCard/LargeCardGutter.jsx]: otherwise `undefined` ends up in the node `class="..."`

Comment (ggascoigne) [reply src/components/native/LargeCard/LargeCardGutter.jsx]: do you just want to use classNames here as well?

Comment (andrewleyva) [reply src/components/native/LargeCard/LargeCardGutter.jsx]: 👍

Comment (cdaringe) [reply src/components/native/LargeCard/LargeCardGutter.jsx]: yep.  subsequent PR!

**Pull request merged**